internal/access/destination.go

```diff
@@ -27,20 +27,12 @@ func SaveDestination(c *gin.Context, destination *models.Destination) error {
 }
 
 func GetDestination(c *gin.Context, id uid.ID) (*models.Destination, error) {
-	db, err := RequireInfraRole(c, models.InfraAdminRole, models.InfraViewRole, models.InfraConnectorRole, models.InfraUserRole)
```

Comment:
Previously every identity was granted `InfraUserRole`, so this statement would be a no-op unless someone had deleted the grant for a particular user (which I assume would have caused other problems).

Now we simply allow any authenticated user to perform these two operations, which should be the same as before.